doc/release/1.14.0-notes.rst

```diff
@@ -193,19 +193,42 @@ raising a ``TypeError``.
 -----------------------------------------------------------------
 When indexed with a float, the dtype object used to raise ``ValueError``.
 
-Changes to array printing, and the new "legacy" printing mode.
---------------------------------------------------------------
-The new ``sign='-'`` option (see new features below) causes the ``repr`` of
-float arrays to often omit a whitespace previously printed in the sign
-position, and 0d arrays also now print with small whitespace and precision
-differences (see changes below). These changes are likely to break downstream
-user's doctests.
+Many Changes to array printing, and the new "legacy" printing mode.
+-------------------------------------------------------------------
+The ``str`` and ``repr`` of ndarrays and numpy scalars have been changed in
+a variety of ways. These changes are likely to break downstream user's
+doctests.
 
 These new behaviors can be disabled to mostly reproduce numpy 1.13 behavior by
 enabling the new 1.13 "legacy" printing mode. This is enabled by calling
 ``np.set_printoptions(legacy="1.13")``, or using the new ``legacy`` argument to
 ``np.array2string``, as ``np.array2string(arr, legacy='1.13')``.
 
+In summary, the major changes are:
+
+ * The ``repr`` of float arrays often omits a whitespace previously printed 
+   in the sign position. See the new ``sign`` option to ``np.set_printoptions``.
+ * Floating-point arrays use a new algorithm for their decimal representations,
+   allowing floats to be represented "uniquely".  This means ``float16`` arrays
+   will print with fewer digits. ``float64`` arrays should be unaffected. See
+   the new ``floatmode`` option to ``np.set_printoptions``.
+ * Floating-point scalars now print with exactly the number of digits needed to
+   be uniquely specified, so ``float16`` will be shorter and ``float128`` 
+   longer. The ``str`` of ``float64`` scalars is no longer truncated in python2.
+ * ``NaT`` values in datetime arrays are now properly aligned
+ * ``MaskedArray`` arrays have a new single-line output mode for short arrays
```

Comment:
This isn't true - they always wrap to another line. What changed is they:
* Show the dtype
* Include commas
* Correctly align wrapped arrays